lib/static/components/section/section-common.js

```diff
@@ -76,19 +84,11 @@ export class SectionCommon extends SectionBase {
 
 export default connect(
     ({view: {expand, viewMode, filteredBrowsers}, suites}, ownProps) => {
-        const origSuite = suites[ownProps.suiteId];
-        let suite;
-
-        if (filteredBrowsers.length > 0) {
-            suite = filterBrowsers(cloneDeep(origSuite), filteredBrowsers);
-        } else {
-            suite = origSuite;
-        }
-
         return {
             expand,
             viewMode,
-            suite
+            suite: suites[ownProps.suiteId],
+            filteredBrowsers
```

Comment:
"filteredBrowsers" may be removed from this line if SectionCommon (lib/static/components/suites.js) will be created with "filteredBrowsers" as property. look at "filterByName".
there will be less confusion.